src/jit/inlinepolicy.cpp

```diff
@@ -26,7 +26,19 @@
 
 InlinePolicy* InlinePolicy::GetPolicy(Compiler* compiler, bool isPrejitRoot)
 {
-    // For now, always create a Legacy policy.
+
+#ifdef DEBUG
+
+    // If JitInlineRandomSeed is set, use the RandomPolicy.
+    unsigned seed = JitConfig.JitInlineRandomSeed();
+    if (seed != 0)
+    {
+        return new (compiler, CMK_Inlining) RandomPolicy(compiler, isPrejitRoot, seed);
+    }
```

Comment:
How about Random policy by default uses method hash as the seed?  Specifying an additional random seed is optional.  It is an additional control if one wants to use.